packages/opc-header/README.md

```diff
@@ -0,0 +1,46 @@
+# This is opc-header 👋
+
+Web component based on lit-element
```

Comment:
Header Web component based on lit-element, it can be used in Angular,React, other framwork, plain javascrip and html like  other element of HTML example <opc-header> </opc-header>

is it okay ?